src/store/actions.js

```diff
@@ -24,25 +24,27 @@ export default {
     const balance = await state.sdk.balance(state.account.publicKey).catch(() => 0);
     commit('updateBalance', convertToAE(balance));
   },
-  async fetchTransactions({ state }, { limit, page, recent }) {
+  async fetchTransactions({ state, getters }, { limit, page, recent }) {
     if (!state.middleware) return [];
     const { publicKey } = state.account;
     let txs = await Promise.all([
       state.middleware.getTxByAccount(publicKey, { limit, page }),
-      (async () =>
-        (
-          await axios
-            .get(
-              `${BACKEND_URL}/cache/events/?address=${publicKey}&event=TipWithdrawn${
-                recent ? `&limit=${limit}` : ''
-              }`,
-            )
-            .catch(() => ({ data: [] }))
-        ).data.map(({ address, amount, ...t }) => ({
+      (async () => {
+        const transactionsFromEvents = await fetchJson
```

Comment:
Why don't do it without the intermediate variable as it was before? Probably, using `then` would be more readable